components/service/glean/README.md

```diff
@@ -124,6 +127,33 @@ the following command can be used:
 
 `adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean.debug.GleanDebugActivity --ez logPings true`
 
+### Debug intent receiver
```

Comment:
Mh, how is this mechanism different than the one above? Do we prefer this one for some reason? If so, we can get rid of the other one, or mention when to use one or the other.

Reply:
I mentioned this in the PR comment.  I think we should prefer using intents because it doesn't require restarting the app which may cause unnecessary side effects when trying to debug.

Reply:
Ouch, sorry, I should have been more explicit, it's been a hard morning :) I mean, the current approach doesn't require an app restart as well - did you try and it worked differently?

Reply:
It looked like that to me.  I'll double-check, though.  If it doesn't cause a restart, I totally agree there's no need to using a different approach and I'll refactor this PR to use `Activity`.

Reply:
You're right -- it doesn't seem to cause a restart.  I think I was testing it wrong before.